src/map/mob.cpp

```diff
@@ -5096,7 +5096,7 @@ uint64 MobDatabase::parseBodyNode(const ryml::NodeRef& node) {
 				continue;
 			}
 
-			if (!CHK_RACE2(constant)) {
+			if (constant <= RC2_NONE || constant >= RC2_MAX) {
```

Comment:
I thought we wanted RC2_NONE support for import?
If you want to make a difference between import and normal check it with the "exists" variable.

Reply:
I misinterpreted as said above mb
```
			bool active;

			if (!this->asBool(raceNode, raceName, active))
				return 0;

			if (active)
				mob->race2.push_back(static_cast<e_race2>(constant));
			else
				util::vector_erase_if_exists(mob->race2, static_cast<e_race2>(constant));
		}
```

Reply:
Ah, fine then!